src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpMtlsCertificateManager.cs

```diff
@@ -0,0 +1,297 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET
+
+using System.Net.Security;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+
+/// <summary>
+/// Manages certificate loading, validation, and security checks for mTLS connections.
+/// </summary>
+internal static class OtlpMtlsCertificateManager
+{
+    internal const string CaCertificateType = "CA certificate";
+    internal const string ClientCertificateType = "Client certificate";
+    internal const string ClientPrivateKeyType = "Client private key";
+
+    /// <summary>
+    /// Loads a CA certificate from a PEM file.
+    /// </summary>
+    /// <param name="caCertificatePath">Path to the CA certificate file.</param>
+    /// <returns>The loaded CA certificate.</returns>
+    /// <exception cref="FileNotFoundException">Thrown when the certificate file is not found.</exception>
+    /// <exception cref="InvalidOperationException">Thrown when the certificate cannot be loaded.</exception>
+    public static X509Certificate2 LoadCaCertificate(string caCertificatePath)
+    {
+        ValidateFileExists(caCertificatePath, CaCertificateType);
```

Comment:
Out of curiosity, have you tried testing with any web-based applications like ASP.NET Core, and does it use the hard-coded file path from this location?